fighthealthinsurance/common_view_logic.py

```
@@ -238,6 +239,9 @@ def create_or_update_appeal(
         company_fax_number: str = "415-840-7591",
         pubmed_ids_parsed: Optional[List[str]] = None,
         pending: Optional[bool] = None,
+        fax_pwyw: Optional[int] = None,
```

Ruff (0.14.6)

Unused method argument: `fax_pwyw`

(ARG002)